src/pages/components/menu-buttons/usage.mdx

```diff
@@ -138,10 +138,19 @@ A. Icon button <br /> B. Menu <br />
 
 #### Height
 
+Menu buttons come in three sizes: small, medium, and large. See more about size
+specifications on the [Style](/components/menu-buttons/style/) tab. When using a
+menu button trigger, the height of the menu item (singular) reflects the size of
+the button being used.
+
+| Size   | Height (px / rem) |
+| ------ | ----------------- |
+| Small  | 32 / 2            |
+| Medium | 40 / 2.5          |
+| Large  | 48 / 3            |
+
 Menu items come in four sizes: extra small, small, medium, and large. See more
```

Comment:
The paragraph information in this section seems a little repetitive and staggered. I think it could all be consolidated above the table instead of having different sections of body copy. You could try something like this: 
<img width="563" alt="Screenshot 2025-02-03 at 1 54 34 PM" src="https://github.com/user-attachments/assets/d404b0ea-86b0-4db3-af9c-c9113ffb6743" />